README.md

```diff
@@ -49,9 +49,40 @@ _tick_ is used for many industrial applications including:
 
 ## Installation
 
-### Requirements
+### Windows
 
-_tick_ currently works only on Linux/OSX systems and requires Python 3.4 or newer. Please have the required Python dependencies in your Python environment:
+  Requries:
+- python 3.4+
+- swigwin 3.0+
+- Microsoft Visual C++ (https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2017)
+
+HowTO:
+
+    Install python 3 
+    Download and extract swigwin
+    Add install directories to PATH environment variable
+    Download wheels from http://www.lfd.uci.edu/~gohlke/pythonlibs:
+    - numpy MKL
+    - numpy-doc
+    - scipy
+
+
+run:
+
+    pip install wheel
+    pip install /path/to/numpy-MKL.whl
+    pip install /path/to/numpy-doc.whl
+    pip install /path/to/scipy.whl
+    python -m pip install --upgrade setuptools
+    python setup.py build
+
+Or run the installation script provided if you have cygwin/msys installed:
+    
+    ./tools/windows/install.sh
+
```

Comment:
Sure, I only got it building, I still don't have much of an idea how to use it.  ;)